### ISSUE TITLE
genesis embed the wrong version when running init

### ISSUE THREAD

**Issue by ramonskie** [starkandwayne/genesis#308]
when running
```
genesis init --kit minio --cwd ~/workspace
~/workspace/minio-deployments/.genesis/bin/genesis -v
```
you get an error
```
Can't locate Genesis.pm in @INC (you may need to install the Genesis module) (@INC contains: /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.22.1 /usr/local/share/perl/5.22.1 /usr/lib/x86_64-linux-gnu/perl5/5.22 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl/5.22 /usr/share/perl/5.22 /usr/local/lib/site_perl /usr/lib/x86_64-linux-gnu/perl-base .) at .genesis/bin/genesis line 23.
BEGIN failed--compilation aborted at .genesis/bin/genesis line 23
```
sha1sum `570f78be443dc950bf148b39aba65f0e6a6f988d  .genesis/bin/genesis`

related to https://github.com/starkandwayne/genesis/issues/294

**Comment by dennisjbell** [comment]
This is fixed in #318

**Comment by jhunt** [comment]
#318 has been merged!
